contracts/cw3-fixed-multisig/src/integration_tests.rs

```diff
@@ -0,0 +1,144 @@
+#![cfg(test)]
```

Comment:
If this is meant as a "classic" integration test (i. e. to be compiled to wasm, and run in the VM), it must go in `tests/` directory, and called `integration.rs`. Then you can do `cargo wasm && cargo integration-test` to run it.

Besides, you'll have to modify it to run in the VM context. That's relatively straightforward to do. Take a look at the header comments in any of the integration tests in the `cosmwasm` repo.

Reply:
Another name would be better, but it does belong here. 
It runs in a (mock) multi-contract environment. Actually more useful feedback than vm integration tests in may cases as you capture contract-contract interactions.

No need to move it, maybe just rename it